### ISSUE TITLE
No "object" key in transactions/{hash} output

### ISSUE THREAD

**Issue by richardkiss** [biteasy/blockchain-api#36]
According to https://support.biteasy.com/kb/rest-api/endpoints/transactions-hash there is a field "object" which contains the hex of the transaction. This doesn't seem to exist though.

Example: look at the output of https://api.biteasy.com/v2/btc/testnet/transactions/4586e67ee5adcdbc97ed3d2a026ee8703df2ed3553854c186c216e90cd761b69

**Comment by alolis** [comment]
@richardkiss, yes, there was at initial release but we had to remove it because it was taking too much extra space to store it and there wasn't really a demand for it. I will update the documentation so it won't be confusing.

**Comment by richardkiss** [comment]
Any chance we could get that back in a separate endpoint? (transactionhex or something) I'm assuming you're storing all the transactions raw, so it might not be too difficult.

**Comment by alolis** [comment]
The problem still remains; it takes too much extra disk space to store that info and SSDs cost :)

We do want to include the raw object but it has to wait a bit for now until we finish the hardware upgrades we are doing and maybe then it can be re-added.

I will check the possibility for a separate endpoint and different source as a temporary solution

**Comment by richardkiss** [comment]
I assumed that you were storing the binary and parsing it produce the JSON output, since that would be most space-efficient way. (Of course, if you're not doing it that way, it would be quite a bit of work to switch.)

**Comment by alolis** [comment]
Yes, that would be the most space-efficient way to do it but it would be pointless because we need to be able to filter the stored data, aggregate on them etc etc. I really want to add this again but we need some time to upgrade our hardware first.

May I ask whats the use case for this?

**Comment by richardkiss** [comment]
This is for pycoin, my Python open source bitcoin library that fetches transactions by ID from various services. The easiest way is to just grab the hex and code it. I have code that works most of the time. https://github.com/richardkiss/pycoin/blob/master/pycoin/services/biteasy.py

The code above has a problem with assuming locktime=0 (which is fixed in v2 API) and sequence number for inputs (which I think is not yet fixed).

Since the transaction can be reconstructed with the data you send (most of the time), you could do this yourself and include the hex without storing the binary blob. Note that `json_to_tx` returns a Tx, which has an "as_hex" method.
